### PR TITLE
Create tasks routes/cody

### PR DESCRIPTION
**Pull request by codymjarrett**
Noel, please check the `usersAndTasks.js` file, as well as the `taskscontroller.js` file, specifically **lines 13-23**! Let me know if I'm headed in the right direction with this

### CODE REVIEW

File: routes/api/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const router = require("express").Router();
+const usersAndTasksRoutes = require("./usersAndTasks");
+// const tasksRoutes = require("./tasks");
+
+// API routes
+router.use("/api/users", usersAndTasksRoutes);
```
Comment (noel3740) [inline routes/api/index.js:6]: should be `router.use("/users", users);`

File: routes/api/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const router = require("express").Router();
+const usersAndTasksRoutes = require("./usersAndTasks");
```
Comment (noel3740) [inline routes/api/index.js:2]: You should keep the name of the file as users.js. In the future we might need to add other routes to this file and the name of "usersAndTasks" might not make sense.

**Review by noel3740 — COMMENTED** [review]
Please check my github comments on the api/index.js file.

File: routes/api/index.js (hunk 1 above)
Comment (codymjarrett) [reply routes/api/index.js:6]: OK this makes sense, I overlooked the extra `/api` path.

**Pull request closed without merging**